### ISSUE TITLE
example use code does not work

### ISSUE THREAD

**Issue by gxlarson** [sparkfish/augraphy#185]
The example code listed in the README runs into an error.

This code is:

```
from augraphy import *
pipeline = default_augraphy_pipeline() # line 2
img = cv2.imread("image.png")
data = pipeline.augment(img)
augmented = data["output"]
```

however on line 2, I encounter the following error message:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: default_augraphy_pipeline() missing 1 required positional argument: 'img'
```

**Comment by kwcckw** [comment]
The issue should be resolved now in version 8.0.1. Please reinstall Augraphy and make sure the version is 8.0.1 since it might reinstalling using the cache.

**Comment by kwcckw** [comment]
This should be resolved with this commit: https://github.com/sparkfish/augraphy/commit/8d01a228c9abcf701a7be63d32dc6a195d12f09a

I will proceed to the close the issue and please let me know again if you still facing the same problem.
